app/assets/javascripts/views/story_view.js

```diff
@@ -883,6 +884,29 @@ module.exports = FormView.extend({
     $('.popover').remove();
   },
 
+  backLoggedRelease: function() {
+    let backlogged = false;
+    if(typeof this.model.collection.project.iterations !== 'undefined') {
```

Comment:
```javascript
if(this.model.collation.project.iterations) {
...
}
```

Doesn't work here?

Reply:
It works, using typeof is a more reliable way to check only for undefined, although a bit ugly looking if you ask me.

Reply:
Haha, yeah! I'm going to change it, because... 💅